media-gfx/gif2apng/gif2apng-1.9.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="Converts GIF animations into APNG format"
+DESCRIPTION="This program converts GIF animations into animated PNG format. \
+Usually it makes the files smaller. CLI version is OS-independent."
+HOMEPAGE="http://gif2apng.sourceforge.net/"
+COPYRIGHT="2009-2013 Max Stepin"
+LICENSE="Zlib"
+REVISION="1"
+SOURCE_URI="https://sourceforge.net/projects/gif2apng/files/$portVersion/gif2apng-$portVersion-src.zip"
+CHECKSUM_SHA256="3b21308e935d799b3ffb4a86c6e00ffa4cb9b3f72f52d58d51c66eb0574ae7d2"
+SOURCE_DIR=""
+
+ARCHITECTURES="x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	gif2apng$secondaryArchSuffix = $portVersion
+	cmd:gif2apng$secondaryArchSuffix = $portVersion
```

Comment:
Then we drop the **`$secondaryArchSuffix`** from **`cmd:gif2apng$secondaryArchSuffix`** to let haikuporter know that it should not expect anything in $prefix/bin/x86/, since we're installing to $prefix/bin/.
You should therefore have:
```
	cmd:gif2apng = $portVersion
```